### ISSUE TITLE
NIMROD: Jaws Manager

### ISSUE THREAD

**Issue by kjwoodsISIS** [ISISComputingGroup/IBEX#4112]
As a scientist, I want a jaws manager UI, so that I can control and configure the 6 sets of jaws on NIMROD from a single location.  I also want the configuration of the jaws to be protected, so that visiting scientists cannot change them.

#### Acceptance Criteria
1. The solution must provide equivalent functionality to the existing SECI VI.
1. I can control the configuration of all 6 sets of jaws from the UI.
1. I can control the configuration of all 6 sets of jaws via a genie_python script.
1. The configuration of all 6 sets of jaws must be password protected.

#### Notes
1. The configuration of the jaws on NIMROD is calculated from a projection from the moderator.  Check the SECI VI to see how this calculation is performed and confirm with the scientists.
1. The existing SECI VI is located here: `C:\LabVIEW Modules\Instruments\NIMROD\Jaws`.
1. Information about NIMROD jaws settings is here: `C:\LabVIEW Modules\Instruments\NIMROD\JAW Data`.
1. See also the [NIMROD wiki](https://github.com/ISISComputingGroup/IBEX/wiki/NIMROD-Instrument-Details) page.

**Comment by DominicOram** [comment]
[GUI](https://github.com/ISISComputingGroup/ibex_gui/pull/902)
[Tests](https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/195)
[IOC](https://github.com/ISISComputingGroup/EPICS-motorExtensions/pull/23)
